src/controllers/user.controllers.js

```diff
@@ -33,8 +33,32 @@ export const userLogin = async (data) => {
   const validPass = await bcrypt.compare(data.password, user.password);
   if (!validPass) return { status: 'invalid', message: 'Email or password is wrong' };
 
-  const token = jsonwebtoken.sign({_id:  user._id}, process.env.TOKEN_SECRET)
+  const token = jsonwebtoken.sign({ _id: user._id }, process.env.TOKEN_SECRET);
   response.header('auth-token', token);
 
   return `Witam ${user.name}!`;
 };
+
+export const getById = async (id) => {
+  const user = await User.findById(id).exec();
+
+  if (!user || !user._id) {
+    return { status: 'invalid', message: 'There is no user with this id.' };
+  }
+
+  return { status: 'success', user };
+};
+
+export const getPayment = async (id, price) => {
```

Comment:
How about `deductPayment` or `collectPayment`? The name `getPayment` could suggest the function only fetches unchanged payment.